src/components/ColumnVolumeMarketCap/index.js

```diff
@@ -0,0 +1,35 @@
+import { coinListPercentDisplayColors as colors } from 'utils/constants'
+import { Column } from 'components/Column'
+import { Cell } from 'components/Cell'
+import { shorterNumber } from 'utils/shorterNumber'
+import { formatCurrency } from '@coingecko/cryptoformat'
+import { ColumnPercentBar } from 'components/ColumnPercentBar'
+import { keyGen } from 'utils/keyGen'

```

Comment:
fix the imports

Reply:
Refactored all folders / components / imports to follow this.